### PR TITLE
feat(app): Add the new LHCb run3 geometry into multiple parts

### PR DESCRIPTION
**Pull request by andrewpap22**
## Greetings, 

So after a couple of weeks trying hard to find solution on this, I finally made it and generated multiple .gltf files for the LHCb geometry from the root file and it is even the newly updated run3 LHCb geometry ( exported from gdml to root and then to gltf just a week ago ). 

I will showcase all the steps for this in the upcoming meeting! 

P.S. Oh! And here you can see a screenshot of it working properly! 😄 

pic-selected-210621-1751-10

## Best regards, 

[Andreas](https://andrewpap22.github.io/)

### CODE REVIEW

**Comment by EdwardMoyse** [comment]
Awesome! Nice job @andrewpap22

File: packages/phoenix-ng/projects/phoenix-app/src/app/sections/lhcb/lhcb.component.ts [hunk 1]
```diff
@@ -52,9 +52,100 @@ export class LHCbComponent implements OnInit {
...
+     * The following commented part of code is properly working
+     * and it is the whole new Geometry of the run3 LHCb detector in one single gltf file,
+     * but there is a huge performance issue (max 4fps)
+     * the fault is the root file that was designed with too many subchildren for the parts of
+     * Ecal, Hcal, Muon
+     * and thus the reuslted gltf file generated from the root one has huge performance issues. 
+     * Uncomment the following to see it in action ( you'll have to wait a bit :) )
+    */
+
+    // this.eventDisplay.loadGLTFGeometry(
```
Comment (EdwardMoyse) [inline packages/phoenix-ng/projects/phoenix-app/src/app/sections/lhcb/lhcb.component.ts:65]: Why would you want this if you have the sub-geometries  though?

Comment (andrewpap22) [reply packages/phoenix-ng/projects/phoenix-app/src/app/sections/lhcb/lhcb.component.ts:65]: Yeah, we don't need it, I just left it there for demonstration purposes of the performance that I wanted to do in today's meeting, but then I forgot to remove it from the pull request, thank you for reminding, I'll do it now! :)

**Review by 9inpachi — APPROVED** [review]
Hi Andreas,

Thanks for the amazing work! Looks good to go. ;)

**Pull request merged**